network-api/networkapi/templates/pages/buyersguide/campaign_page.html

```diff
@@ -43,12 +43,8 @@ <h1 class="tw-h1-heading {% if uses_menu %}tw-hidden medium:tw-block{% endif %}"
                             <div class="tw-mb-12 tw-hidden medium:tw-block">
                             </div>
                         {% endblock %}
-                        {% block html_content %}
-                            <div class="cms campaign-page-content wide">
```

Comment:
moved to `network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/campaign_main_body.html`